src/main/java/io/nats/client/impl/Headers.java

```diff
@@ -479,22 +480,21 @@ public ByteArrayBuilder appendSerialized(ByteArrayBuilder bab) {
 	 * @param dest the byte array to write to
 	 * @return the length of the header
 	 */
+	@SuppressWarnings("deprecation")
```

Comment:
No on deprecated methods. I do like the part of assigning the key instead of calling `entry.getKey()` multiple times.